homeassistant/components/sensor/usps.py

```diff
@@ -60,6 +60,7 @@ def __init__(self, session, name):
         self._name = name
         self._attributes = None
         self._state = None
+        self.update()
```

Comment:
@happyleavesaoc, I'm not sure why, maybe calling the update at this point refreshes the cookies or forces a new login, but without it, if the cookie is old, it won't login and the sensors won't show up in the screen at all. Maybe you can suggest a different way to force a login at setup to create a new cookie?